spec/bundler/ui/shell_spec.rb

```diff
@@ -61,4 +61,15 @@
       end
     end
   end
+
+  describe "#tell_err" do
+    context "when stderr is closed" do
+      it "doesn't report anything" do
+        output = capture(:stderr, closed: true) do
+          subject.send(:tell_err, "Something went wrong")
```

Comment:
I wouldn't be testing a private method, instead test against the public API, so the `#error` method in this case.

(make sure to enable `Bundler.settings.temporary(:error_on_stderr => true)`) because `Bundler::UI::Shell` does not print to stderr.